module/ElasticSearch/src/ElasticSearch/VuFind/RecordDriver/ElasticSearch.php

```diff
@@ -38,7 +38,7 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     http://www.vufind.org  Main Page
  */
-class ElasticSearch extends AbstractBase
+abstract class ElasticSearch extends AbstractBase
```

Comment:
Possibly not a good idea ElasticSearch RecordDriver is fallback for not implemented RecordDrivers like document, item, and work.